File: src/models/book.js
class Book {
    constructor(userId, categoryId, ratingId, imageUrl, title, author, finishedOn, pageCount, summary){
        this.userId = userId;
        this.categoryId = categoryId;
        this.ratingId = ratingId;
        this.imageUrl = imageUrl;
        this.title = title;
        this.author = author;
        this.finishedOn = finishedOn;
        this.pageCount = pageCount;
        this.summary = summary;
    }

    id;
    userId;
    categoryId;
    ratingId;
    imageUrl;
    title;
    author;
    finishedOn;
    pageCount;
    summary;
}

export default Book;

File: api/Bookshelf.Core/Models/Domain/Book.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Bookshelf.Core
{
    public class Book
    {
        [Key]
        public int Id { get; set; }
        public int UserId { get; set; }
        public int CategoryId { get; set; }
        public int RatingId { get; set; }
        public string ImageUrl { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public DateTime FinishedOn { get; set; }
        public int PageCount { get; set; }
        public string Summary { get; set; }
    }
}
